### DISCUSSION TITLE
Error of Tooltips in vue js 3 [BUG]

### DISCUSSION THREAD

**Discussion by Oscar963** [tabler/tabler#1862] [Q&A]
### Browser

Chrome

### OS

Windows 10

### Screen size

1920 x 1080

### Describe the bug

Hi.

When i want tu use  the plugins tooltips.js of bootstrap, it doesn't work for me. the popper.js ones don't work either

import { createApp } from "vue";
import App from "./App.vue";
import router from "./router"; // Vue router
import { abilitiesPlugin } from "@casl/vue";
import ability from "./services/ability";
import pinia from "./stores/main"; // Importa la instancia de Pinia

import "@tabler/core/dist/css/tabler.min.css";
import "@tabler/core/dist/js/tabler.min.js";

// Crear una aplicación Vue
const app = createApp(App);

// Usar abilitiesPlugin de CASL con la aplicación Vue
app.use(abilitiesPlugin, ability, { useGlobalProperties: true });

// Integrar la instancia de Pinia con la aplicación Vue
app.use(pinia);

// Usar el enrutador Vue con la aplicación Vue
app.use(router);

// Montar la aplicación en un elemento HTML con id 'app'
app.mount("#app");

### How to reproduce

2. Click on '...'
3. Scroll down to '...'
...

**Comment by LoTwT** [comment]
Please try `vueuse`.

```ts
import { useScriptTag } from "@vueuse/core"

useScriptTag(
  "https://cdn.jsdelivr.net/npm/@tabler/core@1.0.0-beta20/dist/js/tabler.min.js",
)
```

It works for me.
